FunNet/Classes/Multipart/FormDataEncoder.swift

```diff
@@ -58,11 +60,11 @@ public struct FormDataEncoder {
 
 // MARK: Private
 private final class FormDataEncoderContext {
-  var shouldConvertToCamelCase: Bool
+  var keyEncoder: (String) -> String?
```

Comment:
I think we should probably make camelToSnake `(String) -> String` instead of the optional so we can make this string to string. I think if `String(from: data...)` at the end of camelToSnake fails, we should just return the original string